src/MinimapNode.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const MinimapNode = ({ vertices, scalingFactor }) => {
+  const getMinimapNodes = () => {
+    return vertices.map(vertex => ({
+      top: vertex.top * scalingFactor.y,
+      left: vertex.left * scalingFactor.x,
+      width: vertex.width * scalingFactor.x,
+      height: vertex.height * scalingFactor.y,
+    }));
+  };
+
+  const renderNodes = () => {
```

Comment:
No need to make separate function. we can directly return this

Reply:
done